llvm/lib/Support/Unix/Unix.h

```diff
@@ -30,7 +30,10 @@
 #include <cstring>
 #include <string>
 #include <sys/types.h>
+
+#if !defined(__wasi__)
 #include <sys/wait.h>
```

Comment:
Is this header even needed on other platforms? I don't see anything clearly 'wait' related in this header.